src/main/webapp/assets-sc/js/utils.js

```diff
@@ -120,6 +120,19 @@ var validateUniqname = function (value) {
   }
 }
 
+/*used by modal (adding friend)
+  TODO: can we use some standard industrial strength validator
+ */
+var validateEmailAddress = function (value) {
+  var value = $.trim(value);
+  if(value.indexOf('@') !==-1 && value.indexOf('@umich.edu') ===-1  && value.split('@').length === 2 && value.split('@')[0] !== '' && value.split('@')[1] !== ''){
+    return true;
```

Comment:
This may be overkill, but should we also be looking to see that the second half of the address has a '.' for the '.com' or the '.edu'? So for example, we wouldn't want to process people like 'dovek333@mailinator' but 'dovek333@mailinator.com' would be okay. Thoughts?